packages/docs/components/toast.md

```diff
@@ -0,0 +1,374 @@
+::: slot nimatron-all
+
+<ToastPen ref="toastBox"/>
+
+# Toast
+
+Toasts are non-disruptive messaging UIs that appear at the bottom right of the interface to provide quick, at-a-glance feedback on the outcome of an action.
```

Comment:
I strongly believe that toasts should have the flexibility to appear in any direction. This'll significantly expand our use cases for this component over our existing **banner** or **notification** components.

Reply:
The decisions around placement and use cases came out of research done recently and last year. Keeping these messaging components limited in scope is intentional.

Toasts are, by design, not as flexible as something like Callout. We're trying to move in the opposite direction because Design has seen a ton of mixed and inappropriate use of visually similar components.

We do plan on adding other messaging components like Banner/Alert (for always-up messaging) and Callout (for in-page, static messaging).

Reply:
Flexibility in position could lead to inconsistent results, especially if there is the possibility of having multiple present at one time. I'll pose the question of why a designer/developer would choose one corner over the other?

There was also research done that points to the position we've chosen (bottom right) consistently has open space in our UI to not compete with other UIs while allowing the user to read the content on the page. We really considered not competing with the new top bar and left side bar.

Reply:
All these are fair points, but I'm thinking about it from a slightly different perspective - internationalization.

Did we account for RTL languages? While it's common for English and other LTR languages to expect components to appear on the right, this isn't true for all languages we plan to support.

Since this was highlighted as a common goal of Odyssey, we shouldn't handicap ourselves for global expansion this early.

Reply:
Good point. We should keep internationalization support in mind while thinking about how flexible we want our components to be. We are also investing in pseudo localization testing which will stretch out our UI (horizontally and vertically) to see how well it behaves.

Reply:
We don't currently support an `.is-rtl` variant on other components, but it would be easy to handle that via CSS.

Since we don't currently support this or have a way to test it in-situ, I'd prefer punting as opposed to laying out a standard here.

e.g. We don't ship a RTL layout, so without right-hand sidebar support, an RTL Toast would still be covering important UI.